comp/metadata/packagesigning/utils/packagesigningutils_test.go

```diff
@@ -0,0 +1,87 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
```

Comment:
We have a build tag for this test file, but there is no build tag for the file that implements the `ParseRepoFile` function. Is that intended?

Reply:
I only wanted to dismiss running the tests on non-linux platforms. And I need `parseRepoFile` which is used by `inventoryHost` which can run everywhere

Reply:
> And I need parseRepoFile which is used by inventoryHost which can run everywhere

Would that mean that we should be testing this for Windows as well?

Reply:
yes (and there are no repoFiles in windows so the result is predictable)